cni/network/network.go

```diff
@@ -108,8 +109,9 @@ func NewPlugin(name string, config *common.PluginConfig, client NnsClient) (*net
 		return nil, err
 	}
 
+	netlink := netlink.NewNetlink()
```

Comment:
this var shadows the package name. i guess the linter is not run on the target branch?

Reply:
Linter did run on the target branch. Local lint is fine with it too. But do you think this is not a good approach ?

Reply:
@rbtr is this not a warning in the current CI lint setup? @vakalapa yeah, this causes problems. Say you want a second instance of this type in this block using `NewNetlink()`. You won't be able to because the package is shadowed by the variable. Same for any other exported symbol from that package.

Reply:
yeah this is an issue, I don't see any warnings coming from lint but import naming collisions are raised by the ide, ex:

![image](https://user-images.githubusercontent.com/6521405/133474325-37125cb7-c436-40de-a19e-635b4f456a7d.png)

Reply:
current lint config was not flagging shadows of imports, update to include that here: https://github.com/Azure/azure-container-networking/pull/1009

Reply:
Fixed in places where there is a standalone netlink var. This sure will cause confusion later on so keeping it simple.